app/src/main/java/com/framgia/wsm/screen/listrequest/ItemListRequestViewModel.java

```diff
@@ -128,18 +127,26 @@ public String getLeaveTypeCode() {
     }
 
     public String getFromTime() {
-        if (mRequest != null) {
+        if (mRequest != null && mRequest.getCompensation() != null) {
             return mRequest.getCompensation().getFromTime();
         }
         if (mRequestOverTime != null) {
             return mRequestOverTime.getFromTime();
         }
         if (mRequestOff != null) {
-            if(!mRequestOff.getStartDayHaveSalary().getOffPaidFrom().equals("")){
-                return mRequestOff.getStartDayHaveSalary().getOffPaidFrom();
+            if (mRequestOff.getStartDayHaveSalary() != null) {
+                if (!"".equals(mRequestOff.getStartDayHaveSalary().getOffPaidFrom())) {
+                    return mRequestOff.getStartDayHaveSalary().getOffPaidFrom()
+                            + Constant.BLANK
+                            + mRequestOff.getStartDayHaveSalary().getPaidFromPeriod();
+                }
             }
-            if(!mRequestOff.getStartDayNoSalary().getOffFrom().equals("")){
-                return mRequestOff.getStartDayNoSalary().getOffFrom();
+            if (mRequestOff.getStartDayNoSalary() != null) {
+                if (!"".equals(mRequestOff.getStartDayNoSalary().getOffFrom())) {
```

Comment:
WARNING: [CollapsibleIfStatements] These nested if statements could be combined
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/basic.html#CollapsibleIfStatements